### DISCUSSION TITLE
Github Private contributions not showing

### DISCUSSION THREAD

**Discussion by GeorgeCodes17** [community/community#51677] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Select Topic Area

Question

### Body

Hi, I had my GitHub account set to **"Make profile private and hide activity"** but now I have set it to **"Include private contributions on my profile"**.

I have made many commits to GitHub on my own Private repo in Jan & Feb.

- But my contributions are not showing on the front of my GitHub account-> Jan & Feb are empty, any ideas why?
- Is it because I owned the private repo, do they not count?

**Comment by radar-blip** [comment]
Did you remember to `update preferences` after setting to `Include private contributions`?

Also, sometimes its because your email address is not linked to your github acct.
Read more here: [Contributions that are counted](https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-github-profile/managing-contribution-settings-on-your-profile/why-are-my-contributions-not-showing-up-on-my-profile#contributions-that-are-counted)

**Reply by GeorgeCodes17** [reply]
Could it be because...
The commits were made directly to the repo, with no forking?
Or, regarding the section 'In addition, at least one of the following must be true:' Could the problem be that I own the repo myself?

**Comment by MissieKahsay** [comment]
Hi @GeorgeCodes17,
When you make your contributions private, they will not be visible on your public profile page. However, when you change your settings to "Include private contributions on my profile," your private contributions should show up on your public profile, including the contributions you made to your private repository.

However, it's important to note that GitHub only counts contributions made to repositories owned by other users or organizations. Contributions made to your private repositories are not counted towards your public contribution graph.
Best,

**Reply by GeorgeCodes17** [reply]
Yh, all the new commits (last 2 weeks) have been made to a public repo I own

**Reply by airtower-luna** [reply]
> However, it's important to note that GitHub only counts contributions made to repositories owned by other users or organizations.

This is simply wrong. Contributions to your own public repositories always count (as long as they fill the [general requirements](https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-github-profile/managing-contribution-settings-on-your-profile/why-are-my-contributions-not-showing-up-on-my-profile)), for private ones it's subject to the "include private contributions" setting.

**Comment by GeorgeCodes17** [comment]
It is adding my new commits to my contributions, but I cannot see my old commits on the same repo when it was private- for the past 2 months
